src/test/resources/regressions/features/floats/floats-fail-02.gobra

```diff
@@ -0,0 +1,22 @@
+// Any copyright is dedicated to the Public Domain.
+// http://creativecommons.org/publicdomain/zero/1.0/
+
+package floats
+
+//:: ExpectedOutput(postcondition_error)
+ensures res == x - y || res == x * y || res == x / y
+func f(x float32, y float32) (res float32) {
+	return x + y
+}
+
+//:: ExpectedOutput(postcondition_error)
+ensures res != x + y || res != x - y || res != x * y || res != x / y
+func g(x float32, y float32) (res float32) {
+	return x + y
+}
+
+//:: ExpectedOutput(postcondition_error)
+ensures res == y
```

Comment:
would `res == x` go through?

Reply:
(I now looked at the encoding, and the answer should be no, but maybe add a test case, as it should not hold)